cmd/cmd.go

```diff
@@ -126,7 +126,7 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 					} else if err != nil {
 						return err
 					}
-
+					defer f.Close()
```

Comment:
Thanks for the PR, and good catch. Could we instead close the file before the next iteration of the loop?